packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -85,6 +85,15 @@ export interface IContainerRuntimeMetadata {
     readonly gcFeature?: GCVersion;
 }
 
+export interface ICreateContainerMetadata {
```

Comment:
Curious, what's the benefit of having this interface (as opposed to just using IContainerRuntimeMetadata everywhere, with new fields)? I glanced through PR, and I do not see much value in having this separation.

Reply:
See mine and Navin's feedback on earlier iteration:  https://github.com/microsoft/FluidFramework/pull/7527/files/bda30eaf67cdd3641640ae0c550c142154482954#r743162225

Reply:
That's probably fine, but I'll point out that this.createContainerMetadata.summaryCount++ stands out here as not right. I think over time there will be more examples like that, where we do modify metadata, and it's not really "create" metadata any more. Maybe the this.createContainerMetadata should be readonly (all fields) and only changes in constructor, pushing other properties to be changed in another place? It would be bad if that place was - just a bunch of variables...